erpnext/docs/user/manual/en/selling/articles/erpnext-for-services-organization.md

```diff
@@ -1,28 +1,45 @@
-<h1>ERPNext for Service Organizations</h1>
+#ERPNext for Service Organizations
 
-**Question:** At first look, ERPNext looks primarily designed for the traders and manufacturers. Is ERPNext used by service companies as well?
+**Question:** ERPNext looks primarily designed for the traders and manufacturers. Is ERPNext used by companies offering servies?
 
 **Answer:**
-About 30% of ERPNext customers comes from services background. These are companies into software development, certification services, individual consultants and many more. Being into services business ourselves, we use ERPNext to manage our sales, accounting, support and HR operations.
 
-https://conf.erpnext.com/2014/videos/umair-sayyed
+About 30% of ERPNext customers are companies into services. These are companies into software development, certification services, individual consultants and many more. Being into service business ourselves, we use ERPNext to manage our sales, accounting, support and HR operations. Check following video to learn how ERPNext uses ERPNext.
+
+<iframe width="640" height="360" src="//www.youtube.com/embed/b6r7WxJMfFA" frameborder="0" allowfullscreen=""></iframe>
 
 ###Master Setup
 
-Between the service and trading company, the most differentiating master is an item master. While trading and manufacturing business has stock item, with warehouse and other stock details, service items will have none of these details.
+Between the service and trading company, the differentiating master is an **Item** master. Service companies items are non-stock and service items. They don't have stock and warehouse to them.
```

Comment:
Avoid using the term `master`. 

Better:
*The setup for a Service company differs primarily for Items. They don't maintain the Stock for Items and thus, don't have Warehouses.*